src/components/NFTBook/ItemCard.vue

```diff
@@ -286,6 +285,9 @@ export default {
       this.updateNFTOwners();
       try {
         const blockingPromises = [this.fetchISCNMetadata()];
+        if ([PRESET_TYPE.CAMPAIGN, PRESET_TYPE.SHELF].includes(this.preset)) {
+          blockingPromises.push(this.fetchNFTBookPriceByClassId(this.classId));
```

Comment:
Does this throw error if the nft book is not registered on store?

Reply:
Yes, it will throw a `400 CLASS_ID_NOT_FOUND` error.

Reply:
then we should catch it here to not affect other `blockingPromises`